### PR TITLE
Move matching guidance into the main content

### PR DESCRIPTION
**Pull request by bravegrape**
## Background

The matching guidance is currently in the Legacy section, as a result of 
* changing the technical architecture of the connecting process
* changing the structure of the documentation
* the move from the old website 

The decision to have the matching guidance in Legacy was with the understanding it would be move out once a suitable place was found.

## Why are we doing this?

The matching guidance is still useful for services who want to use the identity dataset to match against existing records. This is a common use case.

## How to review

Check that:

* all the relevant information was moved.
* content is still accurate

### CODE REVIEW

**Comment by bravegrape** [comment]
[Move matching guidance to 'Plan your connection'](https://trello.com/c/CQBTscxW/177-move-matching-guidance-to-plan-your-connection)

File: source/plan-your-connection/matching-guidance/create-matching-rules/index.html.md.erb [hunk 1]
```diff
@@ -42,7 +42,7 @@ If you want to reduce the risk of false positives, you might choose to restrict
 
 Phonetic algorithms
```
Comment (chrisholmes) [inline source/plan-your-connection/matching-guidance/create-matching-rules/index.html.md.erb]: I think this is missing a `###`

File: source/plan-your-connection/matching-guidance/matching-dataset/index.html.md.erb [hunk 2]
```diff
@@ -1,33 +1,33 @@
 ---
-title: PID and Matching dataset
+title: PID and user attributes
 weight: 20
 ---
 
-# PID and Matching dataset
```
Comment (chrisholmes) [inline source/plan-your-connection/matching-guidance/matching-dataset/index.html.md.erb]: We try and link to this from other places. Also, maybe move it higher in the list of headings.

File: source/plan-your-connection/matching-guidance/matching-cycles/index.html.md.erb [hunk 3]
```diff
@@ -9,7 +9,7 @@ The matching process consists of 3 matching cycles. The matching service should
```
Comment (chrisholmes) [inline source/plan-your-connection/matching-guidance/matching-cycles/index.html.md.erb]: 'We recommend that the process...'

Switch matching service to service?

File: source/plan-your-connection/matching-guidance/matching-cycles/index.html.md.erb [hunk 4]
```diff
@@ -9,7 +9,7 @@ The matching process consists of 3 matching cycles. The matching service should
 
 If, after 3 cycles, there’s no match, you can decide if you want to create a new account for the user.
 
-The matching service uses limited information from the GOV.UK Verify Hub to run matching cycles. The limited information includes the persistent identifier (PID) and matching dataset.
+The matching service uses limited information from the GOV.UK Verify Hub to run matching cycles. The limited information includes the persistent identifier (PID) and user attributes.
 
 
 ## Cycle 0
```
Comment (chrisholmes) [inline source/plan-your-connection/matching-guidance/matching-cycles/index.html.md.erb]:
```suggestion
## PID Match
```

File: source/plan-your-connection/matching-guidance/matching-cycles/index.html.md.erb [hunk 5]
```diff
@@ -20,11 +20,11 @@ Cycle 0 matches the user’s hashed PID to an existing hashed PID in the local m
 
 ## Cycle 1
```
Comment (chrisholmes) [inline source/plan-your-connection/matching-guidance/matching-cycles/index.html.md.erb]:
```suggestion
## User Attribute Match
```

File: source/plan-your-connection/matching-guidance/matching-cycles/index.html.md.erb [hunk 6]
```diff
@@ -38,16 +38,16 @@ If Cycle 1 finds multiple matches and additional matches are available, continue
 
 Cycle 2 matching uses additional attributes related to identity from another source such as a credit referencing agency or other government department to help the matching process. For example, an additional attribute could be whether the user qualifies for a Blue Badge.
 
-GOV.UK Verify does not currently support Cycle 2 matching. No government connecting service has needed Cycle 2 matching so far. If you think your service needs Cycle 2 matching, [contact the GOV.UK Verify team](LINK).
+GOV.UK Verify does not currently support Cycle 2 matching. No government connecting service has needed Cycle 2 matching so far. If you think your service needs Cycle 2 matching, [contact the GOV.UK Verify team] [support].
 
 ## Cycle 3
```
Comment (chrisholmes) [inline source/plan-your-connection/matching-guidance/matching-cycles/index.html.md.erb]: Use additional attributes

File: source/plan-your-connection/matching-guidance/matching-cycles/index.html.md.erb [hunk 7]
```diff
@@ -38,16 +38,16 @@ If Cycle 1 finds multiple matches and additional matches are available, continue
```
Comment (chrisholmes) [inline source/plan-your-connection/matching-guidance/matching-cycles/index.html.md.erb]: Attributes from another trusted source... match

File: source/plan-your-connection/matching-guidance/index.html.md.erb [hunk 8]
```diff
@@ -1,13 +1,13 @@
...
 
-Your service must be able to handle matching from GOV.UK Verify even if you don’t want to match identities against existing datasource(s) at that time. For example, you might want to [collect the data now](LINKcreatenewaccounts) and use it for matching later or store the information for auditing purposes.
+Your service must be able to handle matching from GOV.UK Verify even if you don’t want to match identities against existing datasource(s) at that time. For example, you might want to collect the data now and use it for matching later or store the information for auditing purposes.
 
 Because there’s no unique identifier for citizens, finding the record involves matching verified information about the user (such as name, address, and date of birth) to a record in a local datastore(s) with a local identifier. For example, if you run an NHS service, the local identifier may be an NHS number.
```
[the remark this answers is not in the document]
Comment (bravegrape) [reply source/plan-your-connection/matching-guidance/index.html.md.erb]: @chrisholmes Do we need the local identifier here? It's not clear what it's saying in this sentence, but are we trying to recommend some best practice?

File: source/plan-your-connection/matching-guidance/index.html.md.erb [hunk 9]
```diff
@@ -19,11 +19,9 @@ These cases can put your service at risk of reputational damage and compliance i
 
 To mitigate these risks, your service must:
 
-- [understand how matching works with GOV.UK Verify](LINKmatchingverify)
-- [evaluate what data sources you have](LINKevaluatedata)
-- [decide your matching strategy](LINKmatchingstrategy)
-- [prepare your local data source(s)](LINKpreparedata)
-- [build a matching service](buildmatchingservice)
+- [evaluate what data sources you have][evaluate-datasources]
+- [decide your matching strategy][matching-strategy]
```
[the remark this answers is not in the document]
Comment (bravegrape) [reply source/plan-your-connection/matching-guidance/index.html.md.erb]: I've read though this page again, and given the narrative, I don't think we need to define what a matching strategy is anywhere on this page.

Plus the term itself is quite descriptive on its own, AND it's linked to a whole page that talks about matching strategies.

File: source/plan-your-connection/matching-guidance/matching-dataset/index.html.md.erb [hunk 10]
```diff
@@ -1,33 +1,33 @@
...
+The Verify Service Provider will hash the PID to make it specific to each service (and meaningless to other services) before sending it on to your service.
 
-The matching dataset contains verified information about a user including their:
+The set of attributes contains verified information about a user including their:
 
 - name
 - address
 - date of birth
 
 It may also include additional data such as historical addresses and gender.
```
[the remark this answers is not in the document]
Comment (bravegrape) [reply source/plan-your-connection/matching-guidance/matching-dataset/index.html.md.erb]: Agreed, but reviewing this page in detail is out of the scope of the current PR.
We'll come back to this when I raise the PR to add content about the structure of the set of user attributes.

File: source/plan-your-connection/matching-guidance/matching-dataset/index.html.md.erb [hunk 11]
```diff
@@ -1,33 +1,33 @@
...
 
-The [eIDAS regulation](LINK), coming into force in late 2018, says that European citizens must be able to use their national electronic IDs to access public services provided by another EU member state.
+The [eIDAS regulation][eidas-regulation] [external link], coming into force in late 2018, says that European citizens must be able to use their national electronic IDs to access public services provided by another EU member state.
 
 In practice, this means European citizens will be able to use their national online identity schemes to confirm their identity and access UK government services.
 
-If your service needs to be able to process EU identities, you need to [configure your MSA](LINK) to do this.
+If your service needs to be able to process EU identities, you need to [configure your VSP][vsp-config] [external link] to do this.
 
 Once configured, both European identities and GOV.UK Verify identities you receive will always be in the format of the [universal JSON matching schema](LINK).
```
[the remark this answers is not in the document]
Comment (bravegrape) [reply source/plan-your-connection/matching-guidance/matching-dataset/index.html.md.erb]: Can't wait 😄

File: source/plan-your-connection/matching-guidance/matching-dataset/index.html.md.erb (hunk 2 above)
Comment (bravegrape) [reply source/plan-your-connection/matching-guidance/matching-dataset/index.html.md.erb]: Let's take some time today to think about the hierarchy we want in the left had nav.

We should also make a list of places that should link to this. Off the top of my head, it should be mentioned on the pages about:

- setting up successful verification
- testing the service can handle the information it gets

File: source/development-steps/index.html.md.erb [hunk 12]
```diff
@@ -30,9 +30,13 @@ You should [use the Compliance Tool to test] that your Verify Service Provider h
 
 Outcome: your Verify Service Provider can consume and produce valid SAML.
 
-For more information, see [How SAML works with GOV.UK Verify][saml].
+Test that you can handle the identities...
```
Comment (chrisholmes) [inline source/development-steps/index.html.md.erb]: Is this right?

**Comment by bravegrape** [comment]
[Re-write 'Development steps'](https://trello.com/c/czlznfJh/350-re-write-development-steps)

**Pull request merged**